src/dataflow/render/mod.rs

```diff
@@ -324,86 +324,92 @@ pub(crate) fn build_dataflow<A: Allocate>(
                             _ => unreachable!("Upsert envelope unsupported for non-Kafka sources"),
                         }
                     } else {
-                        let (stream, capability) = if let ExternalSourceConnector::AvroOcf(c) =
-                            connector
-                        {
-                            // Distribute read responsibility among workers.
-                            let read_style = if c.tail {
-                                FileReadStyle::TailFollowFd
-                            } else {
-                                FileReadStyle::ReadOnce
-                            };
+                        let (stream, capability) =
```

Comment:
The indentation here changing seems to be a bug in rustfmt. The only real change is passing `reader_schema` to `decode_avro_values`.